src/main/java/duke/pack/Event.java

```diff
@@ -0,0 +1,47 @@
+package duke.pack;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected String fullDesc;
+    protected String time;
+    protected LocalDate date;
+
+    /**
+     * creates an event type of task
```

Comment:
A Small thing I realised but some of the description for java docs starts with a capital and some starts with lower case. No biggie, perhaps some consistency. :)